tests/main/views/test_brief_response.py

```diff
@@ -729,6 +729,61 @@ def test_list_brief_responses_for_brief_id(self):
         assert all(br['briefId'] == another_brief_id for br in data['briefResponses'])
         assert 'self' in data['links']
 
+    def test_list_brief_responses_by_one_framework_slug(self, live_dos2_framework):
+        with self.app.app_context():
+            dos2_brief = Brief(
```

Comment:
Is it worth shifting this out to be a fixture too, alongside the framework?  Seems like having a mix of ways to set up data in one test is a bit weird (but I appreciate we might be doing it that way all over the place), and I thought we're generally trying to use fixtures these days?